pkg/helpers/helpers.go

```diff
@@ -112,3 +114,99 @@ func GetPlatformType(platformType string) (string, error) {
 		return "", errors.New("invalid platform type")
 	}
 }
+
+// The following regular expressions are used in fixLeadingZerosInJSON. They'll be used
+// hundreds of times per verifier run, so we declare them globally to avoid unnecessary
+// recompilation
+var reJSONIntsWithLeadingZero = regexp.MustCompile(`":\s*0+[^,.]+[,}]`)
+var reDigits = regexp.MustCompile(`0*(\d+)`)
+var reBracketedISO8601Timestamps = regexp.MustCompile(`\[[\d-]+T[\d:.]+\]`)
```

Comment:
Minor - I don't think you need to escape the last bracket here.
```suggestion
var reBracketedISO8601Timestamps = regexp.MustCompile(`\[[\d-]+T[\d:.]+]`)
```

Reply:
TIL you don't always have to escape bracket literals in regexes. Your comment also made me realize that I forgot to add unit tests for the function that uses this regex (`RemoveTimestamps`). Both addressed in latest commits. Thanks!